src/components/sequencing/SequenceEditor.svelte

```diff
@@ -265,10 +265,10 @@
     const sequence = viewUpdate.state.doc.toString();
     disableCopyAndExport = sequence === '';
     const tree = syntaxTree(viewUpdate.state);
-    const output = await selectedOutputFormat?.toOutputFormat?.(tree, sequence, commandDictionary, sequenceName);
+    let output = await selectedOutputFormat?.toOutputFormat?.(tree, sequence, commandDictionary, sequenceName);
 
     if ($sequenceAdaptation?.modifyOutput !== undefined && output !== undefined) {
-      $sequenceAdaptation?.modifyOutput(output, parameterDictionaries, channelDictionary);
+      output = $sequenceAdaptation?.modifyOutput(output, parameterDictionaries, channelDictionary);
```

Comment:
Gonna leave a [link](https://github.com/NASA-AMMOS/aerie-ui/pull/1514#discussion_r1809257907) to this conversation thread on the old PR